openshift/aws-account-operator
# Adds random requeue time to credential refresh.

Adds a fuzzing time factor to, over time, randomly distribute credential refreshes.

To test:
* `go test -v github.com/openshift/aws-account-operator/pkg/credentialwatcher`
* Create Account CR or two or Three
* Wait for Credentials to Refresh
* Compare Timestamps, they should no longer be within 59-60 minutes of when they were created or last rotated but within 45-55 minutes, or for the Console URL it should be within 12-15 minutes and not 14-15 minutes.
